src/_tkagg.cpp

```diff
@@ -11,6 +11,21 @@
 #include <Python.h>
 
 #ifdef _WIN32
+#define WIN32_DLL
+#endif
+#ifdef __CYGWIN__
+/*
+ * Unfortunately cygwin's libdl inherits restrictions from the underlying Windows OS,
+ * at least currently. Therefore, a symbol may be loaded from a module by dlsym() only
+ * if it is really located in the given modile, dependencies are not included.
+ * Cygwin does not honor the software accessing underlying WinAPI directly, but we
```

Comment:
Actually https://cygwin.com/cygwin-ug-net/programming.html says "Cygwin allows you to build programs with full access to the standard Windows API"; are the limitations documented somewhere?

Reply:
Interesting, thanks for the link. To tell the truth i don't remember where this comes from. Perhaps even from false memory. I was unable to find any proof of this. OK, i'll reword the comment.